### DISCUSSION TITLE
Team Plan for open source projects

### DISCUSSION THREAD

**Discussion by chandansgowda** [vercel/community#3674] [Help]
### Summary

I'm an open source contributor at [AOSSIE](https://www.aossie.org/). We are building a project which requires us to host a simple API on Vercel. Currently we are on the team plan (since we are an organisation) which is going to expire in 13 days. Is there any way to keep it active ? We are set to go live but if the plan expires in the middle, it'll be a problem. We actually don't want any Team features as of now.

### Steps to Reproduce

Create a new project which is from a Github Organiation

**Comment by Mister-Hope** [comment]
This is the same problem I encountered, Vercel seems to only limit free services to personal, but as any open source project grows, maintainers will eventually create an org and move the project into it, then it will need a team account, thus need payment.

For now, you can try to fock the original repo in your personal account, and use github ci to manualy sync your personal repo on the org repo. That's a solution I finally use.
